runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/excel.ts

```diff
@@ -17,6 +17,11 @@ const excelActionsTranslations = {
     'Process.Details.Modeler.Actions.Excel.FindFirstEmptyRow.Label': 'Find first empty row',
     'Process.Details.Modeler.Actions.Excel.FindFirstEmptyRow.StartColumn.Info': 'Starting column for search. Default A',
     'Process.Details.Modeler.Actions.Excel.FindFirstEmptyRow.StartRow.Info': 'Starting row for search. Default 1',
+    'Process.Details.Modeler.Actions.Excel.InsertColumnsBefore.Label': 'Insert columns before',
+    'Process.Details.Modeler.Actions.Excel.InsertColumns.Column.Info': 'Column name e.g. "AZ" or column number "e.g. "25"',
```

Comment:
redundand " before e.g.

Reply:
imo number without "", so just 25

Reply:
i'd keep instructions and examples together, so 
Column letter or number, e.g. "AZ" or 25

also letter is better 
![MightymikepugMightyMikeGIF](https://github.com/runbotics/runbotics/assets/69820069/e47ab583-6c2a-46da-bd32-e3707be67301)